backend/Tasque.Core.Common/IncomeModels/CreateOrganization.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tasque.Core.Common.IncomeModels
```

Comment:
What do you mean by IncomeModels? I seems to me that it`s not a suitable name

Reply:
Creating Entity creation Models for request body. For example creating different entities we are not setting Id, creation Date etc. To create organization we need only AuthorId and Name, but entity model includes creation date, update date, id, which we are setting on backend, so this types of models helps us to make code more understanable and not to operate with properties, that are unneccesary

Reply:
I understand the general idea but this does not have any relation to the word income
It might be something like PartialModels, or just use DTO

Reply:
Got it, ok

Reply:
I already changed to Incoming* is it ok or still better to rename?

Reply:
it's ok for this PR, we might come up with better name for this namespace later on.